sdk/loadtestservice/azure-developer-loadtesting/swagger/README.md

```diff
@@ -1,20 +1,20 @@
 ## Generate autorest code
 
 ```yaml
-require: https://github.com/Azure/azure-rest-api-specs/blob/9401446f22177696d920cf110893a0de7452ee9e/specification/loadtestservice/data-plane/readme.md
+require: C:/Users/harshanb/source/repos/azure-rest-api-specs/specification/loadtestservice/data-plane/readme.md
```

Comment:
we should only use spec from Azure repo. Make sure you update to Azure repo before merge.